packages/leancode_contracts_generator/lib/src/contracts_generator.dart

```diff
@@ -123,7 +104,9 @@ class ContractsGenerator {
             (l) => l
               ..body.addAll([
                 Code(config.directives),
-                Code("part '${config.name}.g.dart';"),
+                // Only add json_serializable part if there are non-enum members
+                if (db.statements.any((stmt) => !stmt.hasEnum_11()))
```

Comment:
Why? Generated enums use `package:json_serializable#JsonValue`

Reply:
Yes, but if there are **_only_** enums in the base file, nothing is generated